client/me/notification-settings/reader-subscriptions/index.jsx

```diff
@@ -107,7 +107,7 @@ const NotificationSubscriptions = createReactClass( {
 
 						<FormFieldset>
 							<FormLabel htmlFor="subscription_delivery_email_default">
-								{ this.props.translate( 'Default Email Delivery' ) }
+								{ this.props.translate( 'Default email delivery' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.